<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankEditDlgClass</class>
 <widget class="QDialog" name="BankEditDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>175</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>451</width>
    <height>175</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>451</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BankEditDlg</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>帐户代码:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="accountCode"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>注: 10位以内英文或者数字</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>帐户描述:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="accountDescription"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>开户行名称:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bankName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>开户行帐号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bankAccount"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>现金帐户:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="isCash"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>BankEditDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>BankEditDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
